src/main/java/seedu/address/model/timetable/Lesson.java

```diff
@@ -77,6 +77,12 @@ public boolean equals(Object o) {
                 && module.equals(lesson.getModule());
     }
 
+    @Override
```

Comment:
Necessary for hashset removal of similar Lessons, which should be seen as identical to us.

Reply:
Not sure if there are unseen problems from this.

Reply:
Yes, same lessons should have the same hash code, and different lessons should have different hash codes.

Reply:
Usually with hash code, we don’t care so much unless we are creating hash sets or hash maps.

Reply:
Like how Person doesn’t take index into account for both equals and hash code. It doesn’t matter too much.